website/server/libs/slack.js

```diff
@@ -31,6 +32,7 @@ function sendFlagNotification ({
   }
   let titleLink;
   let authorName;
+  let timestamp;
```

Comment:
if you're not changing the value of timestamp it can be `const timestamp = moment...` right when you need it at line 59